web/blog/public-roadmap-and-open-rfcs/index.md

```diff
@@ -0,0 +1,111 @@
+
+3. **Tooling**: What is the right tool to encode the roadmap?
+
+Let's go through them one by one.
+
+Regarding audience, we want to avoid an overly narrow target group, as we are in
+phase of growth where breadth instead of depth is more important. Moreover, we
+gain more transparency if we can unveil all ongoing thrusts. Therefore, we want
+to cover the full spectrum of personas, but make it possible for each individual
+type of persona to get a relevant view.
+
+Regarding interaction, we are actively looking for engagement. Throwing a
+read-only version over the fence to the community is certainly informational,
+but we are looking for creating dialogue. Therefore, we want to allow everyone
+to discuss the various roadmap items in the open.
+
+Regarding tooling, we are in need for something that integrates well with the
+existing environment. Our GitHub presence includes code, documentation, website
+content, and third-party integrations. We also promote use of GitHub Discussions
+to engage with us. This makes GitHub the focal point to engage with the content.
+Therefore, we decided to encode the roadmap as GitHub issues; for clarity in a
+dedicated repository at <https://github.com/tenzir/public-roadmap>.
+
+We decided against dual-purposing the issue tracker of our main repository
+<https://github.com/tenzir/vast> because it would add roadmap items as many
+open, long-running issues that scatter the attention and potentially confuse the
+community. That said, the primary value of the issue tracker is the layer on top
+of issues: [Projects][github-projects], which allows for organizing issues
+across multiple dimensions in a visually appealing way. Here's how it looks as
+of today:
+
+[![Github Roadmap](github-roadmap.jpg)][roadmap]
+
+[github-projects]: https://docs.github.com/en/issues/planning-and-tracking-with-projects
+
+## Open RFCs
+
+The roadmap provides a lens into the short-term future. We don't want it to be
+just read-only. Fundamentally, we want to build something that our users love.
+We also want to tap into the full potential of our enthusiasts by making it
+possible to engage in technical depth with upcoming changes. Therefore, we are
+establishing a formal [Request for Comments (RFC) process][rfc].
+
+To get an idea, how an RFC looks like, here's the [RFC template][rfc-template]:
+
+[rfc-template]: https://github.com/tenzir/vast/blob/master/rfc/000-TEMPLATE.md
+
+import CodeBlock from '@theme/CodeBlock';
+import Template from '!!raw-loader!@site/../rfc/000-TEMPLATE.md';
+
+<CodeBlock language="markdown">{Template}</CodeBlock>
+
+[RFC-001: Composable Pipelines](https://github.com/tenzir/vast/pull/2511) is an
+example instantiation of this template.
+
+When it comes to reviewing RFCs, we face biased governance as of today, given
+that Tenzir develops VAST. Rather than the Tenzir's engineering making acceptance
+decisions, we'd rather have an independent governing board. Until then, we
+perform all reviews fully in the open.
```

Comment:
I would want to tone this down a bit. There will always be (strategic) discussions between Tenzir employees that are not fully in the open, and these have an influence that may not necessarily be open.

It is important that the decision-making stays with us in the foreseeable future, but we want to give the community a way to have a direct influence on the it.

Reply:
I gave it a different spin, please take another look.

Reply:
Yes, this reads more nicely. Thanks!